source/numerics.tex

```diff
@@ -8959,174 +8959,174 @@
 
 namespace std {
   constexpr @\placeholder{floating-point-type}@ acos(@\placeholder{floating-point-type}@ x);
-  constexpr float acosf(float x);
-  constexpr long double acosl(long double x);
+  constexpr float               acosf(float x);
```

Comment:
Do we need some italics correction here to compensate for the italics introduced by the `\placeholder`?

Reply:
![image](https://github.com/cplusplus/draft/assets/22040976/1a2e4c06-6dd1-410b-976f-a2773389a0f3)

Yes, the italic font looks to be a few pixels misaligned here.